lib/src/build/polymer_smoke_generator.dart

```diff
@@ -223,11 +223,13 @@ class PolymerSmokeGenerator {
   ///    * invoke methods in event handlers.
   _processClass(ClassElement cls, Recorder recorder) {
     if (!_hasPolymerMixin(cls)) return;
+    if (cls.node is! ClassDeclaration) return;
```

Comment:
It can now return an EnumDeclaration or ClassDeclaration technically. We need it at least for now.